src/js/components/landing/recentActivity/DeleteModal.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+  * DeleteModal.jsx
+  * Created by Minahm Kim 03/10/17
+  **/
+
+import React from 'react';
+import Modal from 'react-aria-modal';
+import { hashHistory } from 'react-router';
+import * as Icons from '../../SharedComponents/icons/Icons.jsx';
+
+import * as ReviewHelper from '../../../helpers/reviewHelper.js';
+
+export default class DeleteModal extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			errorMessage: "",
+			disable: false
+		};
+	}
+
+	clickedDeleteButton() {
+		ReviewHelper.deleteSubmission(this.props.id)
+			.then((data) =>{
+				if(data.message == 'Success'){
+					this.props.delete();
+
+					// reset the modal if closed
+					this.setState({
+						errorMessage: ''
+					}, () => {
+						this.props.closeModal();
+					});
+				}
+			})
+			.catch((error) => {
+				this.setState({
+					disable: true,
+					errorMessage: (error.httpStats === 400 || error.httpStatus === 403) ? error.message : "An error occurred while attempting to delete the submission. Please contact your administrator for assistance."
+				});
+			});
+	}
+
+	closeModal(e) {
+		if (e) {
+			e.preventDefault();
+		}
+
+		// reset the modal if closed
+		this.setState({
+			errorMessage: ''
+		}, () => {
+			this.props.closeModal();
+		});
+	}
+
+	render() {
+		let hideClose = "";
```

Comment:
If hideClose never changes you can remove it from here and, in turn, simplify the className section of the div it's associated with.